packages/inference/test/HfInference.spec.ts

```diff
@@ -778,7 +778,7 @@ describe.concurrent("HfInference", () => {
 	describe.concurrent(
 		"Fal AI",
 		() => {
-			const client = new HfInference(env.HF_FAL_KEY);
```

Comment:
use `??` instead of `||` (same for others)

Reply:
resolved in https://github.com/huggingface/huggingface.js/pull/1226/commits/664a38a9ec80101cefd08d79d5600008f46fdb36